### ISSUE TITLE
Blocked time can be counted twice

### ISSUE THREAD

**Issue by lopperman** [lopperman/jira-issue-analysis#2]
If there are multiple methods used to mark an issue as blocked, blocked time is getting double-counted for any overlapping timespan.

- e.g. Flagged field with 'impediment' value 
- e.g. Priority field with 'blocked' value

**Comment by lopperman** [comment]
wrote code below to adjust/remove duplicate blockers (`AnalyzeIssues.cs`)

```

        private void RemoveBlockerOverlap(IssueCalcs ic)
        {
            var bList = ic.Blockers.OrderBy(x=>x.StartDt).ToList();
            foreach (var b1 in bList)
            {                   
                if (b1.Removed == false && bList.Any(x=>x.tmpID != b1.tmpID && x.StartDt <= b1.EndDt && x.Removed == false))
                {   
                    if (!b1.EndDt.HasValue) {b1.EndDt = DateTime.Now;}
                
                    var overlaps = bList.Where(x=>x.tmpID != b1.tmpID && x.StartDt <= b1.EndDt).ToList();
                    foreach (var o1 in overlaps)
                    {
                        if (!o1.EndDt.HasValue) {o1.EndDt = DateTime.Now;}
                        o1.Adjusted = true;
                        var newStartDt = b1.EndDt.Value.AddSeconds(1);
                        o1.AdjustmentNotes.Add($"Overlap - StartDt Changed from '{o1.StartDt}' to '{newStartDt}'");
                        o1.StartDt = newStartDt;
                        if (o1.EndDt.Value <= o1.StartDt)
                        {
                            o1.Removed = true;
                        }
                    }

                }
            }
            var goodList = bList.Where(x=>x.Removed == false).ToList();
            ic.Blockers = goodList;
        }

```

**Comment by lopperman** [comment]
fix will be included in 0.3.5
